### DISCUSSION TITLE
How to .addEventListener scroll or use onscroll ?

### DISCUSSION THREAD

**Discussion by datntfe** [glideapps/glide-data-grid#790] [Q&A] [answered]
I try get position of scrollbar, but them not working. 

I need use it for "guide tour" and use it switch tabs to another 'grid' how to keep screen view position after back

**Comment by jassmith** [comment, marked as answer]
You're looking for `onVisibleRegionChanged`

**Reply by datntfe** [reply]
Thank you !!!!!
